app/themes/dashboard/ui/styles/theme.less

```diff
@@ -43,3 +43,11 @@ z-index: 10;
 .pre-xml {
   background-color: #ecf0f5;
 }
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
```

Comment:
Please remove the rounded corners. This design is supposed to have square buttons.